### DISCUSSION TITLE
Export personal vault ?

### DISCUSSION THREAD

**Discussion by kramlech** [dani-garcia/vaultwarden#1719] [Q&A]
Hello everybody

Do you know if it is possible to use the CLI to export a personal vault?
(You can do it in Bitwarden with the command "bw export").

Thank you

**Comment by BlackDex** [comment]
you gave the command your self already

**Reply by kramlech** [reply]
How to use this command with docker, please ? I don't find the bw command in the container...

**Reply by jjlin** [reply]
It's not part of the container. See

* https://bitwarden.com/help/article/cli/
* https://bitwarden.com/help/article/change-client-environment/#cli
